index.js

```diff
@@ -137,6 +138,19 @@ function report (times) {
   }))))
 }
 
+const argv = yargs
+  .usage('Usage: $0 [options]')
+
+  .boolean('production')
+  .describe('production', 'Skip devlopment dependencies (devDependencies)')
+
+  .boolean('no-optional')
+  .describe('no-optional', 'Skip optional dependencies (optionalDependencies)')
+
+  .help('help')
+  .alias('h', 'help')
+  .argv
```

Comment:
I prefer putting this up at the top, and also using `var` instead of `const`, but love adding `-h` and `--help`!